guessit/rules/properties/streaming_service.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+streaming_service property
+"""
+from rebulk import Rebulk
+from rebulk.rules import Rule, RemoveMatch
+
+from ...rules.common import seps
+from ...rules.common.validators import seps_surround
+
+
+def streaming_service():
+    """Streaming service property.
+
+    :return:
+    :rtype: Rebulk
+    """
+    rebulk = Rebulk().string_defaults(ignore_case=True)
+    rebulk.defaults(name='streaming_service', validator=seps_surround)
+
+    rebulk.string('AE', value='A&E')
+    rebulk.string('AMBC', value='ABC')
+    rebulk.string('AMZN', value='Amazon Prime')
+    rebulk.string('AS', value='Adult Swim')
+    rebulk.string('iP', value='BBC iPlayer')
```

Comment:
This one is interesting. All the others have a convention:
- Up to 4 letters (Usually all uppercase) 
- before `format`
- after `screen_size` whenever possible

This one is 5 letters and not uppercase. Anyway, that's just a convention, not a rule